src/streaming/metrics/utils/ManifestParsing.js

```diff
@@ -88,10 +88,16 @@ function ManifestParsing (config) {
                             return;
                         }
 
-                        for (const prop in reporting) {
-                            if (reporting.hasOwnProperty(prop)) {
-                                reportingEntry[prop] = reporting[prop];
-                            }
```

Comment:
Question here: Why do we remove the more generic parsing in favor of parsing for explicit values?

Reply:
The generic parsing was originally done this way so that no profile-specific (eg DVB) Reporting stuff existed outside of the self-contained metrics reporting extensions ie when you choose to build without that stuff.

I have no real feeling either way on whether that was the right idea or not but it was a requirement at the time.

Reply:
Definitely not a necessity, the desired effect would be the same. I tried to make changes based on the existing code base so I looked at how BaseURL is handled (as it also has DVB profile-specific attributes), which [parses the DVB attributes explicitly](https://github.com/Dash-Industry-Forum/dash.js/blob/4961649e62505bd16e9aabb3acc1f689577ba048/src/dash/models/DashManifestModel.js#L986). Happy to change to what would be the preference for handling profile-specific stuff.